test/helpers/cilium.go

```diff
@@ -333,6 +349,96 @@ func (c *Cilium) ReportFailed(commands ...string) {
 		fmt.Fprint(wr, res.Output())
 	}
 	fmt.Fprint(wr, "StackTrace Ends\n")
+	c.ReportFailedFull()
+}
+
+//ReportFailedFull runs a variety of commands and write the results to
+//testResultsPath
+func (c *Cilium) ReportFailedFull() {
+	reportCommands := map[string]string{
+		"sudo cilium endpoint list -o json": "endpoint_list_txt",
+		"sudo cilium service list -o json":  "service_list.txt",
+		"sudo cilium config":                "config.txt",
+		"sudo cilium bpf lb list":           "bpf_lb_list.txt",
+		"sudo cilium bpf ct list global":    "bpf_ct_list.txt",
+		"sudo cilium bpf tunnel list":       "bpf_tunnel_list.txt",
+		"sudo cilium policy get":            "policy_get.txt",
+		"sudo cilium status":                "status.txt",
+	}
+
+	reportEndpointsCommands := map[string]string{
+		"cilium endpoint get %s":    "endpoint_%s.txt",
+		"cilium bpf policy list %s": "bpf_policy_list_%s.txt",
+	}
+
+	testPath, err := ReportDirectory()
+	if err != nil {
+		c.logCxt.Errorf("Can't create Test results path '%s' err '%s'", testPath, err)
```

Comment:
Do not use upper-case within error messages. C.f. https://github.com/golang/go/wiki/CodeReviewComments#error-strings

For consistency, please use "cannot" instead of "can't". This will make searching logs easier.

Also, you can just do `c.logCxt.Error("<text>: %s", err) instead of writing "err" in the log message itself. Golang errors are usually implicitly chained by colons.